### PR TITLE
Fix minor layout inconsistencies

### LINKED ISSUE #104: Minor layout inconsistencies
**Description**
1. 
There are inconsistencies when marking invalid user input. E.g. in multisig simple and scheduled transfers, an invalid memo only shows a red error message but the memo characters stay blue. The invalid amount turns red, not only a red error message.
coloring

In single sig transfers, the characters become read as well.
coloring_2

2.
Wrong spelling is indicated but DW does not seem to allow the usual right-click context menu, which would allow auto correction.
spelling

**Versions**
- dw 1.2.0 beta2
- Windows
- Chrome

### PR DESCRIPTION
**Pull request by shjortConcordium**
## Purpose

Fix #104

## Changes

Make memo text error-colored on error in multisig flow.
Add context menu that allows cut/copy/paste and spellchecking.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.

### CODE REVIEW

File: app/main/menu.ts [hunk 1]
```diff
@@ -81,3 +82,49 @@ export function createMenu(window: BrowserWindow) {
...
+                enabled: params.editFlags.canPaste,
+            },
+        ]);
+
+        if (params.dictionarySuggestions.length > 0) {
+            menu.append(new MenuItem({ type: 'separator' }));
+        }
+
+        // Add each spelling suggestion
+        for (const suggestion of params.dictionarySuggestions) {
```
Comment (soerenbf) [inline app/main/menu.ts:103]: How many are suggestions are included? Could it be possible that the list could be too long, making the context menu huge, or is this limited by electron or on OS level?

Comment (shjortConcordium) [reply app/main/menu.ts:103]: We should check it on each OS, but this is how electron does it in their own documentation:
https://www.electronjs.org/docs/tutorial/spellchecker

Comment (shjortConcordium) [reply app/main/menu.ts:103]: On linux it does seem to never exceed 5 suggestions.

Comment (soerenbf) [reply app/main/menu.ts:103]: I haven't had it exceed 5 either on macOS (tried 3 times), maybe it's fine then 🙂

**Review by soerenbf — APPROVED** [review]

**Pull request merged**